packages/asc-ui/src/components/Footer/FooterContent/FooterContentStyle.ts

```diff
@@ -9,5 +10,10 @@ export default styled.div<Props>`
   display: flex;
   flex-direction: column;
   margin-left: ${({ indent }) => (indent ? '28px' : '0')};
+  max-width: 303px;
```

Comment:
plz remove? very specific, it's on the grid right?

Reply:
this was requested from design. set on 300px